reference/provider-cli.md

```diff
@@ -70,7 +70,7 @@ In order to change a particular setting \(for eg. price settings\) type:
 
 You can also combine multiple settings in one command as follows:
 
-`golemsp settings set --cpu-per-hour 3 --cores 7` which will change your NGNT per hour to "3" and adjust the numbers of shared cpu cores to "7".
+`golemsp settings set --cpu-per-hour 3 --cores 7` which will change your tGLM per hour to "3" and adjust the numbers of shared cpu cores to "7".
```

Comment:
i think this should be GLM

- `tGLM` is speciically for rinnkeby only GLM
- `GLM` is specifically or mainnet AND all generic cases